lib/diagnostics.py

```diff
@@ -112,48 +112,32 @@ def get_debian_version():
     else:
         return 'Unable to get Debian version.'
 
-
-def get_raspberry_code():
-    """
-    Temporary workaround and just all on another function
-    """
-    return raspberry_pi_helper.parse_cpu_info().get('revision', False)
-
-
-def get_raspberry_model(raspberry_pi_revision):
-    """
-    Quick DRY workaround. Needs to be refactored later.
-    """
-    return raspberry_pi_helper.lookup_raspberry_pi_revision(raspberry_pi_revision).get('model', False)
-
-
-def get_raspberry_revision(raspberry_pi_revision):
-    """
-    Quick DRY workaround. Needs to be refactored later.
-    """
-    return raspberry_pi_helper.lookup_raspberry_pi_revision(raspberry_pi_revision).get('revision', False)
-
+"""
+@TODO
+Need to find different methods (built-in) that does not rely on manually entered static database info to get these results.
+Example, for mem: cat /proc/meminfo for memory
+Not sure how we should approach this for now, but not really important at the moment.
 
 def get_raspberry_ram(raspberry_pi_revision):
-    """
-    Quick DRY workaround. Needs to be refactored later.
-    """
     return raspberry_pi_helper.lookup_raspberry_pi_revision(raspberry_pi_revision).get('ram', False)
 
-
 def get_raspberry_manufacturer(raspberry_pi_revision):
-    """
-    Quick DRY workaround. Needs to be refactored later.
-    """
     return raspberry_pi_helper.lookup_raspberry_pi_revision(raspberry_pi_revision).get('manufacturer', False)
```

Comment:
Remove this call to `lookup_raspberry_pi_revision`.
You can also remove the function definition for `get_raspberry_manufacturer` since this method is not being used anymore (upon `grep`-ping on the source code).